### ISSUE TITLE
Can we call custom pdf plugin in bootstrap plugin?

### ISSUE THREAD

**Issue by Gurushesh-Metapercept** [infotexture/dita-bootstrap#140]
while generating html5 output using html5-bootstrap plugin can I call custom pdf plugin and generate 2 outputs in same destination? 

if possible, then how can I archive this.

**Comment by jason-fox** [comment]
You wouldn't do this directly using the bootstrap plugin.  The way to do this is to make sequential calls to build HTML and PDF outputs, and define the relevant parameters using  [project files](https://www.dita-ot.org/dev/topics/using-project-files)

Alternatively if your project contains multiple deliverables, you can pass the `--deliverable` option to generate output for a single deliverable ID.

```console
dita --project=all.xml --deliverable=pdf
```

There is also the possibility to integrate your output generation using [Github Actions](https://www.dita-ot.org/dev/topics/using-github-actions) - this could create HTML, create PDF and push them both to a web-server. You can see a same workflow [here](https://github.com/jason-fox/dita-ot-plugins/blob/master/.github/workflows/ci.yml) with the result [here](https://github.com/jason-fox/dita-ot-plugins/actions/runs/5976751447/job/16215269839).

In summary, you'll need some underlying DITA-OT call that creates the PDF, some underlying DITA-OT call that creates the HTML, and something to aggregate the outputs in the correct location - how you achieve this with your infrastructure is  up to you, and not defined within an individual plugin.
